Multi-Region Serverless (#16926)
cockroachdb/docs: _includes/v23.1/ui/databases.md

| Indexes                        | The number of indexes in the table.                                                                      |
{% if page.cloud != true -%}
| Regions                        | The regions and nodes on which the table data is stored. This is not displayed on a single-node cluster. |
{% else %}

> | Regions                        | The regions on which the table data is stored.

Reviewer: ```suggestion
| Regions                        | The regions where the table data is stored.
```
Is this a reason this is a little different from line 25?

- **Auto Stats Collection**: Whether [automatic statistics collection]({{ link_prefix }}cost-based-optimizer.html#table-statistics) is enabled.
{% if page.cloud != true %}
- **Regions/Nodes**: The regions and nodes on which the table data is stored. This is not displayed on a single-node cluster.
{% else %}

> - **Regions**: The regions on which the table data is stored.

Reviewer: At this point I wonder if we should reuse this fragment to keep it in sync more easily across 4 different lines.
